dotCMS/src/main/java/com/dotcms/integritycheckers/IntegrityChecker.java

```diff
@@ -68,12 +68,13 @@
      * When there is conflicts, this method will fix them depending of the
      * implementation of every checker
      * 
-     * @param endpointId
-     *            Server identifier were we need to fix conflicts
+     * @param key
+     *            JWT Token key when using JWT token in Push Publish or
+     *            Server identifier were we need to fix conflicts when not using JWT token
      * @throws DotDataException
      * @throws DotSecurityException
      */
-    public void executeFix(final String endpointId) throws DotDataException, DotSecurityException;
+    public void executeFix(final String key) throws DotDataException, DotSecurityException;
```

Comment:
Codacy found an issue: [Unnecessary modifier 'public' on method 'executeFix': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190239)